src/main/java/net/sf/jabref/specialfields/SpecialFieldsUtils.java

```diff
@@ -162,8 +163,7 @@ public static void syncSpecialFieldsFromKeywords(BibEntry be, NamedCompound ce)
         if (!be.hasField(KEYWORDS_FIELD)) {
             return;
         }
-        Set<String> keywordList = net.sf.jabref.model.entry.EntryUtil
-                .getSeparatedKeywords(be.getField(KEYWORDS_FIELD));
+        Set<String> keywordList = net.sf.jabref.model.entry.EntryUtil.getSeparatedKeywords(be);
```

Comment:
I also prefer that, but I am quite sure that someone else did indeed change it to look like this...